tests/js/spec/stores/organizationStore.spec.jsx

```diff
@@ -68,4 +70,41 @@ describe('OrganizationStore', function() {
       dirty: false,
     });
   });
+
+  it('loads in sorted teams', async function() {
+    const organization = TestStubs.Organization();
+    OrganizationActions.update(organization);
+    // wait for action to get dispatched to store
+    await tick();
+
+    const teamA = TestStubs.Team({slug: 'a_team'});
+    const teamB = TestStubs.Team({slug: 'b_team'});
+    const teams = [teamB, teamA];
+    TeamActions.loadTeams(teams);
+    // wait for action to get dispatched to store
+    await tick();
+
+    // verify existence and sorted order of loaded teams
+    expect(OrganizationStore.get().organization.teams).toEqual([teamA, teamB]);
+  });
+
+  it('loads in sorted projects', async function() {
+    const organization = TestStubs.Organization();
+    OrganizationActions.update(organization);
+    // wait for action to get dispatched to store
+    await tick();
```

Comment:
Same as above.

Reply:
You might need two ticks to get reflux to consistently be populated.

Reply:
I see, does reflux usually take two ticks to dispatch an action?

Reply:
I think dispatching needs a tick and then the store's trigger needs a tick

Reply:
Oh if I don't need the trigger as I'm only testing the state of the store (not the state of the listening components) do I still need both ticks? Or are they necessary so that the next `await tick()` will await the correct thing, and not just the trigger?

Reply:
Probably not, we can deal with it if we see it being flakey.